phpcs.xml.dist

```diff
@@ -77,6 +77,7 @@
 	<!-- Exclude PHPUnit tests from file and class name sniffs (for Core parity). -->
 	<rule ref="WordPress.Files.FileName.NotHyphenatedLowercase">
 		<exclude-pattern>/phpunit/*</exclude-pattern>
+		<exclude-pattern>*.asset.php</exclude-pattern>
```

Comment:
See https://github.com/WordPress/gutenberg/pull/59705/files#r1517560039. I think that would work 😄